File: src/ayd1ndemirci/EconomyAPI/command/SeeMoneyCommand.php
<?php

namespace ayd1ndemirci\EconomyAPI\command;

use ayd1ndemirci\EconomyAPI\Economy;
use pocketmine\command\Command;
use pocketmine\command\CommandSender;
use pocketmine\player\Player;
use pocketmine\Server;
use pocketmine\utils\TextFormat;

class SeeMoneyCommand extends Command
{

    public Economy $economy;

    public function __construct(Economy $economy)
    {
        $this->economy = $economy;
        parent::__construct(
            "seemoney",
            "See money to player"
        );
        $this->setPermission("pocketmine.group.user");
    }

    public function execute(CommandSender $sender, string $commandLabel, array $args)
    {
        if (!$sender instanceof Player) return;

        if (empty($args[0])) {
            $sender->sendMessage(TextFormat::RED . "/seemoney player");
            return;
        }
        $player = Server::getInstance()->getPlayerByPrefix($args[0]);
        if ($player instanceof Player) {
            $sender->sendMessage(sprintf(TextFormat::GREEN .  "%s's money is %s", $player->getName(), $this->economy->getManager()->getPlayerMoney($player->getName())));
        } else $sender->sendMessage(sprintf(TextFormat::RED . "The player named %s could not be found", $args[0]));
    }
}

File: src/main/java/ayd1ndemirci/EconomyAPI/command/SeeMoneyCommand.java
package ayd1ndemirci.EconomyAPI.command;

import cn.nukkit.command.Command;
import cn.nukkit.command.CommandSender;
import cn.nukkit.command.data.CommandParamType;
import cn.nukkit.command.data.CommandParameter;
import ayd1ndemirci.EconomyAPI.api.EconomyAPI;
import ayd1ndemirci.EconomyAPI.manager.Manager;

public class SeeMoneyCommand extends Command {

    public SeeMoneyCommand() {
        super("seemoney", "Oyuncunun parasına bakarsınız", "/seemoney <playerName>");
        commandParameters.clear();
        this.commandParameters.put("default", new CommandParameter[]{
                CommandParameter.newType("player", true, CommandParamType.TARGET)
        });
        setPermission(Manager.SEE_MONEY);
    }

    public boolean execute(CommandSender sender, String label, String[] args) {
        if (!testPermission(sender)) return false;
        if (!sender.getServer().isOp(sender.getName())) return false;
        if (args.length < 1) {
            sender.sendMessage(getUsage());
            return false;
        }
        String playerName = String.join(" ", args);
        if (sender.getServer().getPlayer(playerName) != null) playerName = sender.getServer().getPlayer(playerName).getName();
        sender.sendMessage(String.format("§2» %s §aisimli oyuncunun parası: §2%s", playerName, EconomyAPI.formatMoney(EconomyAPI.getMoney(playerName))));
        return true;
    }
}